config/docker/qemu/Dockerfile

```diff
@@ -10,4 +10,5 @@ RUN apt-get update && \
 	qemu-system-s390x \
 	qemu-system-sparc \
 	qemu-system-x86 && \
+	qemu-utils \
```

Comment:
```suggestion
	qemu-system-x86 \
	qemu-utils && \
```

Reply:
Didn't push all changes, thanks for spotting. Corrected.